@xen-orchestra/web/src/types/xo/vm-template.type.ts

```diff
@@ -0,0 +1,28 @@
+import type { XoPool } from '@/types/xo/pool.type'
+import type { XoVbd } from '@/types/xo/vbd.type.ts'
+import type { XoVif } from '@/types/xo/vif.type.ts'
+import type { Branded } from '@core/types/utility.type'
+
+type Disk = {
+  bootable: boolean
+  device: string
+  size: number
+  type: string
+  SR: string
+}
+
+export type XoVmTemplate = {
+  $VBDs: XoVbd['id'][]
+  $pool: XoPool['id']
+  CPUs: { max: number; number: number }
+  VIFs: XoVif['id'][]
+  boot: { firmware: string; order: string }
+  id: Branded<'vm-template'>
+  uuid: Branded<'vm-template'>
```

Comment:
`id` and `uuid` aren't the same.
the type of `uuid` is `string`
```suggestion
  id: Branded<'vm-template'>
  uuid: string
```